### DISCUSSION TITLE
How to update portainer in Alpine-Docker LXC?

### DISCUSSION THREAD

**Discussion by johndoe0815** [tteck/Proxmox#1759] [Help Needed] [answered]
Hello,
I get a notification in portainer about an update.
`bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/alpine-docker.sh)"
` in the console only updates docker. 
In found this discussion https://github.com/tteck/Proxmox/discussions/485 and tried the commands that are mentioned, but like someone also writes there, it doesn't seem to work in the alpine setup. I still get the update notification.
![grafik](https://github.com/tteck/Proxmox/assets/92877294/90ea6d5f-824f-42b2-b907-c8e5f90ae1ab)

How can portainer be updated in there?

**Comment by therealsebo** [comment, marked as answer]
Portainer itself is just a docker-container. So you have to pull a newer version of portainer to make the update. 
To update the container you can follow the instructions on https://docs.portainer.io/start/upgrade/docker

**Comment by shinyone** [comment]
I created a script here:   https://github.com/tteck/Proxmox/discussions/485#discussioncomment-8009996   since I had a problem with the port number (double check in your case)
